v0/sources/iterable/transform.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const fs = require("fs");
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./mapping.json"), "utf-8")
+);
+const Message = require("../message");
+// const { mappingJson } = require("./mapping");
+
+function process(event) {
+  const message = new Message(`Iterable`);
+
+  // event type is always track
+  const eventType = "track";
+
+  message.setEventType(eventType);
+
+  message.setEventName(event.eventName);
+
+  message.setProperties(event, mapping);
```

Comment:
we should use `setPropertiesV2` and use the updated mapping

Reply:
Done